lib/chef/resource/chef_client_config.rb

```diff
@@ -250,20 +250,21 @@ def string_to_symbol(prop_val)
         introduced: "17.8"
 
       action :create, description: "Create a client.rb config file for configuring #{ChefUtils::Dist::Infra::PRODUCT}." do
```

Comment:
This description should probably be updated now since it now does more (the resource itself now does more as well)

Reply:
added config file and folders to description.